default.nix

```diff
@@ -91,6 +91,12 @@ rec {
     inherit haskell webCommon webCommonPlutus;
   });
 
+  # Cardano
+  inherit (plutus.haskell.project.hsPkgs.cardano-cli.components.exes) cardano-cli;
```

Comment:
Can we put this in `nix/pkgs/default.nix`? That already has a bunch of executables pulled out from haskell packages defined there.